optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md

```diff
@@ -17,20 +17,24 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
-## Changes in supported environments
+## Changes in behaviour
 
 ### Telemetry
 
-Optimize no longer gathers telemetry data. This change includes removing it from UI and ElasticSearch. The associated key in configuration file (`telemetry.telemetryEndpoint`) was removed.
+Optimize no longer gathers telemetry data. This change includes removing it from UI and ElasticSearch. The associated key in the configuration file (`telemetry.telemetryEndpoint`) was removed.
```

Comment:
Perhaps link to the docs on telemetry here?

Reply:
There is no page to link to because it gets deleted with this migration, I think. Note that this section is not new, migration was deleted before